test/runner/requirements/constraints.txt

```diff
@@ -17,3 +17,4 @@ ntlm-auth >= 1.0.6 # message encryption support
 requests-ntlm >= 1.1.0 # message encryption support
 requests-credssp >= 0.1.0 # message encryption support
 voluptuous >= 0.11.0 # Schema recursion via Self
+botocore >= 1.10.0 # adds support for the following AWS services: secretsmanager, fms, acm-pca, neptune
```

Comment:
Not necessary to add this here. This will mean people not running AWS tests will also install botocore. There's an AWS requirements file which lists boto3, which also installs botocore.

Reply:
Understood! That line has been removed.